src/components/Button.js

```diff
@@ -0,0 +1,10 @@
+import React from 'react';
+import PropType from 'prop-types';
+
+const Button = ({ children }) => (
+  <button type="button">{children}</button>
+);
```

Comment:
- Also here please let's stay within the realms of what the project requirement asks, in this case:
![image](https://user-images.githubusercontent.com/46115725/93668225-c48b1b80-fa50-11ea-8c8b-490db57d22ae.png)
- Incoming propr should be titled buttonName or bName or something along those lines would be acceptable.

Reply:
changed "children" prop to "buttonName" as suggested